test/specs/scenarios/document.js

```diff
@@ -174,18 +174,36 @@ export const documentScenarios = async (lang) => {
         confirm.verifyCheckReadabilityMessage(copy)
       })
 
-      it('should return file size too large message for doc', async () => {
+      it('should return file size too large message for PDF document upload', async () => {
         goToPassportUploadScreen(
           driver,
           welcome,
           documentSelector,
           `?language=${lang}`
         )
         documentUpload.clickUploadButton()
-        uploadPassportImageFile('over_10mb_face.jpg')
```

Comment:
My only concern is that this MR adds 24MB to the repository that will have to be downloaded by everyone working on it. Maybe instead you can generate these files on the fly before tests?

Reply:
Interesting idea. Let's chat to find out more about how to go about doing this. :)

Reply:
> 24MB to the repository that will have to be downloaded by everyone

peanuts in my opinion. 24MB that you clone once. Not really a big deal.
If still is an issue you can always do git-lfs.

I don't think it pays off to generate them on the fly just to save 24MB in git repo

Reply:
After discussion, it looks like generating images on the fly will take a long time so not an ideal trade off solution unfortunately. But I have reduced this test file's size down to 10.5MB with compression.